src/frontend/sharedComponents/LeaveProjectModalContent/AlreadyOnProject.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import ErrorIcon from '../../images/Error.svg';
+import {defineMessages, useIntl} from 'react-intl';
+import {BottomSheetModalContent} from '../BottomSheetModal';
+
+const m = defineMessages({
+  leaveProj: {
+    id: 'screens.LeaveProject.AlreadyOnProject.leaveProj',
+    defaultMessage: 'Leave Current Project',
+  },
+  goBack: {
+    id: 'screens.goBackect.AlreadyOnProject.goBack',
+    defaultMessage: 'Go Back',
+  },
+  alreadyOnProject: {
+    id: 'screens.goBackect.AlreadyOnProject.alreadyOnProject',
+    defaultMessage: 'You are already on a project',
+  },
+  onProject: {
+    id: 'screens.goBackect.AlreadyOnProject.onProject',
+    defaultMessage: 'You are on {projectName}',
+  },
+  leaveWarning: {
+    id: 'screens.goBackect.AlreadyOnProject.leaveWarning',
```

Comment:
nit: is `goBackect` intentional?

Reply:
https://github.com/digidem/comapeo-mobile/pull/526/commits/4b9e23254fa5ab53c2b0cd9ac0d1a90a5432c675